docs/release_process.rst

```diff
@@ -0,0 +1,165 @@
+.. _release_process:
+
+===============================
+Release process for *libCellML*
+===============================
+
+This document details the process for making a release of *libCellML*.
+Releases are made using builders from the Buildbot Continuous Integration (CI).
+There are four steps in making a release.
+
+1. `Step 1 - Setting the version number`_
+2. `Step 2 - Preparing the release`_
+3. `Step 3 - Creating the release`_
+4. `Step 4 - Finalising the Release`_
+
+Each section has further details on what actions are required for a particular step.
+Each step must be done in order from step 1 through to step 4.
+
+For all the steps in creating a release, you must be logged in to the Buildbot CI and be in the *admin* group.
+If you are neither of these you will not be able to perform any of these actions.
+
+.. note::
+
+  Merging in pull requests when a release is underway is not recommended and more importantly has not been tested.
+  To determine if a release is underway check the repository for the presence of a branch named *version_change* or *release_staging_<version number>*.
+
+Step 1 - Setting the version number
+===================================
+
+The version number for the project can be set using the *Set version Builder* (:numref:`libcellml_release_process_set_version_builder`).
+The *Set version Builder* sets the version that is entered into the interface, it does not increment the version.
+The version that you set in the interface will be applied as is to the codebase.
+
+.. figure:: ./images/release_process/set_version_builder.png
+   :align: center
+   :alt: Buildbot set version builder.
+   :name: libcellml_release_process_set_version_builder
+
+   Set version builder on Buildbot.
```

Comment:
Minor: shouldn't this be called "Set Version Builder" rather than "Set version Builder", i.e. capitalise "Version"?